docs/documentation/head/connect.md

```diff
@@ -74,6 +74,13 @@ Connection conn = DriverManager.getConnection(url);
 
 	The database user's password. 
 
+* **service** = String
```

Comment:
Is it ok for the documentation ?

Reply:
I think so. There's no reason to copy the documentation over